### PR TITLE
FIX: displays user-card when clicking on avatar of post small action

### PR DESCRIPTION
**Pull request by jjaffeux**

### CODE REVIEW

File: app/assets/javascripts/discourse/widgets/post-small-action.js.es6 [hunk 1]
```diff
@@ -102,7 +102,7 @@ export default createWidget("post-small-action", {
     contents.push(
       avatarFor.call(this, "small", {
         template: attrs.avatar_template,
-        username: attrs.avatar,
```
Comment (eviltrout) [inline app/assets/javascripts/discourse/widgets/post-small-action.js.es6]: Wow, that's a weird bug. How long has that been there?

Comment (jjaffeux) [reply app/assets/javascripts/discourse/widgets/post-small-action.js.es6]: Don't you remember doing it 3 years ago ?

screenshot 2019-01-24 at 23 01 29

:trollface:

Comment (jjaffeux) [reply app/assets/javascripts/discourse/widgets/post-small-action.js.es6]: In fact it was probably working when you did it maybe ? maybe `attrs.avatar` was different at that time ?

Comment (eviltrout) [reply app/assets/javascripts/discourse/widgets/post-small-action.js.es6]: Totally possible.

**Pull request merged**